### PR TITLE
fix(styles): correct styling for user select in nav

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-548

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by CITguy** [comment]
### BEFORE
account-users-before

### AFTER
account-users-after

File: src/components/rxForm/rxForm.less [hunk 1]
```diff
@@ -200,28 +200,6 @@
...
-    top: 5px;
-  }
-}
-
-.account-users {
-  float: right;
-  box-sizing: border-box;
-  width: 50%;
-}
-
```
Comment (CITguy) [inline src/components/rxForm/rxForm.less]: Moved to `rxApp.less` where all other navigation styles are defined.

**Comment by lfielder** [comment]
Design LGTM

**Comment by 100stacks** [comment]
LGTM

**Comment by bry4n** [comment]
LGTM

**Pull request merged**